### PR TITLE
Added Dockerfile of maurorigo

### PR DESCRIPTION
**Pull request by maurorigo**

### CODE REVIEW

File: ciao.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Ciao
```
Comment (luca-heltai) [inline ciao.md:1]: Could you remove this file?

File: tests/rigo/ciao.m [hunk 2]
```diff
@@ -0,0 +1 @@
+Ciao
```
Comment (luca-heltai) [inline tests/rigo/ciao.m:1]: And also this?

File: tests/rigo/test.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+print("Test code using numpy and jax with cpu.")
+
+arr = np.arange(10)
+
+arrnp = np.random.permutation(arr)
+print(f"Permuted with numpy: {arrnp}.")
+
+key = jax.random.PRNGKey(13)
+arrjnp = jax.random.permutation(key, arr)
+print(f"Permuted with jax: {arrjnp}.")
```
Comment (luca-heltai) [inline tests/rigo/test.py:14]: Can you add an assert that checks some conditions? In this test, technically, you are not testing anything, just running jax. 

Make sure, for example, that the `arrjnp` has the right size (just to make sure your jax installation is not doing anything stupid).

**Review by luca-heltai — CHANGES_REQUESTED** [review]

File: tests/rigo/ciao.m (hunk 2 above)
Comment (maurorigo) [reply tests/rigo/ciao.m:1]: Oh I'm sorry, I forgot my pull request was still open, those files weren't supposed to end up in it.

File: tests/rigo/test.py (hunk 3 above)
Comment (maurorigo) [reply tests/rigo/test.py:14]: Sure! Thank you

**Comment by maurorigo** [comment]
Note: this was my original pull request when we just had to push a working Dockerfile, it is not yet the one with github actions. For that one, I'm using a different branch, I've updated the image and I've added more tests. I may open the pull request within the weekend. The mess comes from the fact that I created this pull request from my main branch.

**Comment by maurorigo** [comment]
I've now noticed that with the last commit I would be deleting the contents of tests/heltai (is this actually the case?). I wanted to try to fix this, so I fetched and pulled upstream main on my local main. The problem now is that my local main branch doesn't show the "tests" folder, but it keeps saying I'm up to date with upstream main. I've also tried to merge my main with the commit where the contents of tests/heltai were added, but it still tells me I'm up to date without showing the "tests" folder. What should I do?

**Comment by luca-heltai** [comment]
Close this pull request, checkout a new branch from your main, fetch upstream, and rebase on upstream, i.e.,:

```
git checkout -b rigo-docker
git fetch upstream
git rebase upstream/master
```
if you have conflicts during this process, then abort the rebase
```
git rebase --abort
```
and do
```
git reset upstream/master
git add docker/maurorigo
git commit -m 'Added docker file.'
```
the reset is not a hard reset, so it will maintain your changes, and show the diffs between your branch and master. Drop any changes unrelated to `docker/maurorigo`. :)

**Comment by maurorigo** [comment]
Alright, thank you!

**Pull request closed without merging**